_format/1.1/index.md

```diff
@@ -2371,6 +2331,8 @@ An error object **MAY** have the following members:
     exists; if it doesn't, the client **SHOULD** simply ignore the pointer.
   * `parameter`: a string indicating which URI query parameter caused
     the error.
+  * `profiles`: an array of the requested profile URIs that the server was not
```

Comment:
First reaction: Clever addition!

Second reaction: I wonder whether the combination of `links.type = https://jsonapi.org/errors/profile-not-supported` (already required per one of the first change hunks) plus a `links.profiles` would not be adequate too? That would avoid adding an error object member.
Put differently: why an error object member and not an error object's `links` object member?

Reply:
This has been removed.